scripts/cleanup-dev-resources.sh

```
@@ -0,0 +1,102 @@
+fi
+
+# 스냅샷 정리 (7일 이상 된 것만)
+echo "🔍 오래된 EBS 스냅샷 검색 중..."
+CUTOFF_DATE=$(date -d '7 days ago' -u +%Y-%m-%dT%H:%M:%S.000Z)
+SNAPSHOTS=$(aws ec2 describe-snapshots \
+    --owner-ids self \
+    --filters "Name=tag:AutoCleanup,Values=true" \
+    --query "Snapshots[?StartTime<='$CUTOFF_DATE'].SnapshotId" \
+    --output text)
+
+if [ ! -z "$SNAPSHOTS" ]; then
```

Shellcheck (0.10.0)

[style] Use -n instead of ! -z.

(SC2236)